README.md

```diff
@@ -247,3 +254,20 @@ first part of the URL path, e.g.:
   "/otherdomain.biz": "http://10.0.1.4:5555",
 }
 ```
+
+## Using Redis
+
+If you require multiple instances of CHP to be running and kept in sync, you can use [redis] to to store the route table
+rather than the default in memory option.
+
+To do so, you'll need to pass in a couple of options to CHP when launching it.  Specifically, you'll need to set the
```

Comment:
To configure CHP to launch with redis as the external storage provider, run the following:

```bash
configurable-http-proxy --storage-provider redis
```

This command launches CHP and uses [redis] on `localhost:6379 and the default redis database.